package.json

```diff
@@ -39,7 +39,7 @@
     "typescript": "^5.0.2"
   },
   "dependencies": {
-    "@axiom-crypto/halo2-js": "0.1.75",
+    "@axiom-crypto/halo2-js": "0.1.79",
```

Comment:
do you think we should make it "^0.1.79"? I think there shouldn't be (m)any breaking changes going forward.

Reply:
yeah i think that should be okay